packages/gatsby/src/utils/webpack.config.js

```diff
@@ -763,6 +763,38 @@ module.exports = async (
     }
   }
 
+  if (stage === `build-javascript` || stage === `build-html`) {
+    const cacheLocation = path.join(
+      program.directory,
```

Comment:
Could we add these directories as ones that are excluded from the automatic cache clearing?

Reply:
They fall into what `PRESERVE_WEBPACK_CACHE` already does preserve:
https://github.com/gatsbyjs/gatsby/blob/257207696fec45c4d1bed2f55644d8389f6a9ad6/packages/gatsby/src/services/initialize.ts#L422-L425

So I would say we could work toward making that flag stable and default